.github/workflows/doc.yml

```diff
@@ -0,0 +1,31 @@
+name: PBxplore's Documentation
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8]
```

Comment:
Why not also 3.6 here ?

Reply:
For the build of documentation, I don't think it's necessary to test against all versions of Python.
From what I read, Readthedocs use python 3.7 to produce the documentation. I just add 3.8 for the future but I could have just test 3.7

Reply:
I see, thanks for the answer